src/coprocessor/mod.rs

```diff
@@ -117,7 +117,13 @@ impl ReqContext {
         txn_start_ts: Option<u64>,
     ) -> Self {
         let deadline = Deadline::from_now(max_handle_duration);
-        let bypass_locks = TsSet::new(context.take_resolved_locks());
+        let bypass_locks = TsSet::new(
+            context
+                .take_resolved_locks()
+                .into_iter()
+                .map(Into::into)
+                .collect(),
+        );
```

Comment:
How about keep `TsSet`'s inner Vec<u64> unchanged, and do the conversion in `TsSet`'s `contains` method? Otherwise, this conversion seems causes an extra allocation.

Reply:
I've used transmute.